src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -319,6 +320,8 @@ def load_arguments(self, _):
         c.argument('enable_managed_identity', action='store_true')
         c.argument('assign_identity', type=str,
                    validator=validate_assign_identity)
+        c.argument('disable_local_accounts', action='store_true')
+        c.argument('enable_local_accounts', action='store_true')
```

Comment:
Why do we need two parameters with opposite meanings? Isn't one parameter already able to support the current feature?